packages/runtime/test/consumption/notifications.test.ts

```diff
@@ -122,12 +135,12 @@ describe("Notifications", () => {
             expect(result).toBeAnError("'LocalNotification' not found.", "error.transport.recordNotFound");
         });
 
-        test("wrong message id for sentNotification", async () => {
+        test("wrong message id for saveSentNotification", async () => {
```

Comment:
- "Message ID" instead of "message id".
- Rephrase: "saveSentNotification" to "save Notification with status 'Sent'"